packages/omgidl-serialization/src/MessageReader.ts

```diff
@@ -106,6 +108,7 @@ export class MessageReader<T = unknown> {
     options: HeaderOptions,
   ): Record<string, unknown> {
     const readMemberHeader = options.usesMemberHeader && deserInfo.usesMemberHeader;
+    const readDelimiterHeader = options.usesDelimiterHeader && deserInfo.usesDelimiterHeader;
```

Comment:
is there a semantic difference between how you're using `uses-` and `read-` prefixes? Or should I think of them as both meaning the same thing, like "delimiter header is [expected to be] present"?